guides/slack.md

```diff
@@ -0,0 +1,67 @@
+# Slack
+
+## Purpose of this guide
+
+As of writing, we've grown to over 150 team members and we're also operating almost fully remotely. As such, Slack is one of our most important cross company tools and used extensively by everyone.
+This guide is not intended to be a set of rules around using slack, this document is a guide to how we use slack. This should help everyone understand how to get the most out of slack while also being able to minimise distractions, notification noise and control their own focus time, without missing important updates from their team and the wider organisation.
+
+# Profiles
+
+Please complete your profile fully on joining slack - this means a suitable profile picture (it does not have to be a picture of you, but a clear picture of your face is ideal), your full name and a representative display name.
+
+You can optionally fill in the pronunciation guide for your name and add your preferred pronouns if you feel comfortable doing so.
+
+Please also include your up to date email address and group/role, and the location you are based.
+
+# Channels
+
+Slack channels broadly fall under the following categories:-
+
+ - Project channels (named after the project). If you are working on that delivery, it is expected you'll be in that channel, otherwise open project channels are optional.
+ - Supply channels (prefixed `supply-`). These channels are used by groups focusing on how we _supply_ deliverables. Optional unless directly involved with this group.
+ - Demand channels (prefixed `demand-`). These channels are used by groups focusing on how we engage with other organisations. Optional unless directly involved with this group.
+ - Group channels (such as `ops-` or `academy-`). Channels used by 
+ - Specific purpose channels (usually prefixed with their temporary purpose, for example `bid-`). Temporary channels used for teams fulfilling a specific related task. Optional unless involved with the task, and will be archived after completion.
+ - General channels (no prefix). Company wide chanels such as `announcements`, `help` or regional channels such as `manchester` and _community of practice_ channels prefixed with `cop-`, these channels are used for cross company communication and tend have channel specific usage expecations, which should be detailed in their channel topic.
+ - Off topic channels (prefixed `x-`). These channels are entirely optional to join and use, feel free to create off topic channels if your interest or hobby is not yet represented!
+ - Bot channels or 'firehose' channels. It is occasionally useful to setup bots that post information to specific channels periodically, it is advised that you only need to join these channels if you are involved in monitoring such information.
+ 
+## Mandatory channels
+
+As of writing, it is expected every slack user will be present in `#announcements`, this is so that everyone can be reached in case of an important announcement.
```

Comment:
You can't actually leave `#announcements`, so I think we can talk about channel membership more tersely, maybe even combining it with the section above.

Reply:
Is it worth just being clear about that then?

Reply:
I'm worried about length overall - I don't think there's any harm in including this, but it might cause other things to get missed.